buildbot/master/files/config/master.cfg

```diff
@@ -101,8 +102,10 @@ c['schedulers'].append(schedulers.ForceScheduler(
         "mac-rel-intermittent",
         "mac-rel-wpt1",
         "mac-rel-wpt2",
-        "windows-dev",
-        "windows-nightly",
+        "windows-dev-gnu",
```

Comment:
Please change to `windows-gnu-dev` per https://github.com/servo/servo/pull/14651#issuecomment-268398195, as well as the other instances.